src/server/rpc/libs/RPC

```diff
@@ -12,3 +12,4 @@ nasa.xml	NASA
 twentyquestions.xml	Twenty Questions
 twitter.xml	Twitter
 traffic.xml	Traffic
+nplayer.xml	NPlayer
```

Comment:
It might be nice to name is something more meaningful for kids... Maybe something like "Multiplayer turn helper"